pkgs/stdenv/generic/check-meta.nix

```diff
@@ -125,6 +126,20 @@ let
 
       '';
 
+  remediate_outputs_to_unknown = attrs: let
+      expected_outputs = (attrs.meta or {}).outputsToInstall or [];
```

Comment:
Suggest `attrs.meta.outputsToInstall or []` instead. It will only fail if attrs isn't an attrset or undefined (I think?), and the existing version will as well.

Reply:
```
nix-repl> {}.bar.baz.tux or []
[ ]
```

whoa!

Reply:
Please use camelCase, i.e. `remediateOutputsToUnknown, `expectedOutputs`, and `brokenOutputs`.